notifications_utils/formatters.py

```diff
@@ -619,8 +624,73 @@ def add_language_divs(_content: str) -> str:
 def remove_language_divs(_content: str) -> str:
     """Remove the tags from content. This fn is for use in the email
     preheader, since this is plain text not html"""
-    content = re.compile(FR_OPEN).sub("", _content)
-    content = re.compile(FR_CLOSE).sub("", content)
-    content = re.compile(EN_OPEN).sub("", content)
-    content = re.compile(EN_CLOSE).sub("", content)
+    return remove_tags(_content, FR_OPEN, FR_CLOSE, EN_OPEN, EN_CLOSE)
+
+
+def add_img_tag(_content: str, tag, img_location, alt_text="", height=300, width=300) -> str:
+    """
+    Custom parser to add custom img in the email.
+
+    This is a custom temporary change not meant to exist for more than a few
+    weeks. This should either be removed or upgraded into a full-fledged
+    feature.
+
+    TODO: Review, remove/upgrade this functionality.
+    """
+    ga_seal_regex = re.compile(f"{tag}")  # matches [[tag]]
```

Comment:
nit: `ga_seal_regex` is out of date from previous iteration `tag_regex` maybe? Comment should probably just be `matches tag` as the tag includes `[[ ]]`